editor/sidebar/collaboration/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { PanelBody, PanelRow, FormToggle } from '@wordpress/components';
+
+/**
+ * Internal Dependencies
+ */
+import { collaborationMode } from '../../actions';
+
+function CollaborationPanel( { active = false, ...props } ) {
```

Comment:
@jasmussen had a nicer idea for placing this in an ellipsis menu. (It's ok for an initial pass, though.)

Reply:
Added to the todo list.